source/services/encryption/EncryptionService.stories.tsx

```diff
@@ -110,9 +117,28 @@ const testSymmetricKeySetup = async () => {
   printPublicKeyResult(coApplicantStina, updatedForm);
   await printSymmetricKeyResult(coApplicantStina, updatedForm);
 
-  await setupSymmetricKey(mainApplicantYlva, updatedForm);
+  updatedForm = await setupSymmetricKey(mainApplicantYlva, updatedForm);
   await printSymmetricKeyResult(mainApplicantYlva, updatedForm);
 
+  updatedForm = await encryptFormAnswers(mainApplicantYlva, updatedForm);
+  const encryptedFormCopy = JSON.parse(JSON.stringify(updatedForm));
+
+  const { answers: decryptedAnswers1 } = await decryptFormAnswers(
+    mainApplicantYlva,
+    updatedForm
+  );
+
+  const updatedFormCopy = JSON.parse(JSON.stringify(encryptedFormCopy));
+  const { answers: decryptedAnswers2 } = await decryptFormAnswers(
+    coApplicantStina,
+    updatedFormCopy
+  );
+
+  console.log("encrypted", encryptedFormCopy.answers);
+  console.log("encrypted type:", encryptedFormCopy.encryption.type);
+  console.log("decrypted (applicant)", decryptedAnswers1);
+  console.log("decrypted (co-applicant)", decryptedAnswers2);
+
   console.log("Cleaning up...");
```

Comment:
This feels like a logging that indicates some form of loading behavior but can't see anything that's happening in relation to the logging? So i'm curious to why we have this log here?

Reply:
Yep it's misleading; I'll fix it up!